File: leetcode/0028/strStr.swift
func strStr(_ haystack: String, _ needle: String) -> Int {
    if haystack == needle || needle.count == 0 {
        return 0
    }
    if haystack.count < needle.count{
        return -1
    }
    var a: String = haystack
    for i in 0...(a.count - needle.count) {
        if a.hasPrefix(needle) {
            return i
        }
        a.remove(at: a.startIndex)
    }
    return -1
}

print(strStr("mississippi", "issip"))

File: leetcode/0028/strStr.py
def strStr(haystack: str, needle: str) -> int:
    l_h = len(haystack)
    l_n = len(needle)
    if l_n == 0:
        return 0
    for i in range(l_h - l_n + 1):
        if needle == haystack[i:i+l_n]:
            return i
    return -1

print(strStr("mississippi", "issip"))